README.md

```diff
@@ -22,6 +22,10 @@ A detailed guide for multiple platforms setup could be find [here](https://flutt
 ```
 flutter packages pub run build_runner build
```

Comment:
I think we can add that flag here itself just for the sake of users getting confusion. @aman-singh7 wdy think ?

Reply:
Yep, cool :+1:

Reply:
yes that will be easier for user,  I will make the changes